lib/common/src/RuntimeVersion.php

```diff
@@ -78,7 +78,7 @@ public function currentVersion($options = [])
         $response = $this->remoteRequest->get(self::RUNTIME_METADATA_ENDPOINT);
         $statusCode = $response->getStatusCode();
 
-        if (200 < $statusCode || $statusCode >= 300) {
+        if ($statusCode < 200 || $statusCode >= 300) {
```

Comment:
This bug means that the fallback stylesheet would almost always be used?

Reply:
Yes, we should backport this.

Reply:
Cool. Then I'll squash merge and cherry-pick.